### ISSUE TITLE
[Suggestion] replace crucible and pandora's box images with ones that don't have backgrounds

### ISSUE THREAD

**Issue by HitLuca** [clee/3dprintersforants.com#9]
Just a small suggestion, the main page of this project could look nicer if the crucible and pandora's box printer images had their background removed, to match the rest of the ahestetics

**Comment by clee** [comment]
> Just a small suggestion, the main page of this project could look nicer if the crucible and pandora's box printer images had their background removed, to match the rest of the ahestetics

You're right, that would make them fit the rest of the page better. However, these are the images that the maintainers for Crucible and Pandora's Box provided, so we'd need to convince them to come up with new renders that fit the rest of the site better.
